epochStart/bootstrap/process.go

```diff
@@ -44,6 +44,13 @@ var log = logger.GetOrCreate("epochStart/bootstrap")
 
 const timeToWait = 5 * time.Second
 
+// BootstrapParameters
+type Parameters struct {
```

Comment:
Maybe BootstrapParameters ?

Reply:
it is not recommended to have the same starting string as the package name.

Reply:
`EpochCoordinates` :)

Anyway, fix comment.

Reply:
done